BackEnd/src/main/java/com/unosquare/admin_core/back_end/enums/EventStatuses.java

```diff
@@ -8,7 +8,8 @@
     AWAITING_APPROVAL(1, "Awaiting approval"), //default clientStatus for new holiday requests
     APPROVED(2, "Approved"),
     DENIED(3, "Denied"),
-    CANCELLED(4,"Cancelled");
+    CANCELLED(4,"Cancelled"),
+    REJECTED(5,"Rejected");
```

Comment:
What's the difference between denied and rejected?

Reply:
Makes sense to use Denied instead of a new status. I'll amend that. Thanks.